pkg/distro/fedora/distro.go

```diff
@@ -473,6 +473,7 @@ var defaultDistroImageConfig = &distro.ImageConfig{
 	Timezone:               common.ToPtr("UTC"),
 	Locale:                 common.ToPtr("en_US"),
 	DefaultOSCAPDatastream: common.ToPtr(oscap.DefaultFedoraDatastream()),
+	FirstBootStrategy:      common.ToPtr(osbuild.MachineIdFirstBootYes),
```

Comment:
You lost the conditional use of it `common.VersionGreaterThanOrEqual(d.osVersion, VERSION_FIRSTBOOT)`, but I'm not sure if it was intentional.

Reply:
It was; we considered it a bug (mostly because it was too much work to also turn `defaultDistroConfigs` in Fedora into functions).

Reply:
Now that I look at this; it isn't too much work to change. Do you feel strongly to reintroduce the version guard? :)

Reply:
I don't feel strongly about it. I was just curious...